microsoft-edge/visual-studio/index.md

```diff
@@ -34,52 +34,49 @@ The simplified workflow allows you to debug JavaScript that run in Microsoft Edg
+Select **Script Debugging** > **Enabled**.  
 
 :::image type="complex" source="./media/enable-script-debugging.png" alt-text="Turn on script debugging in Visual Studio" lightbox="./media/enable-script-debugging.png":::
    Turn on script debugging in Visual Studio  
 :::image-end:::  
 
-In the same dropdown, choose **Web Browser** > the preview channel of Microsoft Edge that you want Visual Studio to launch, such as Microsoft Edge Canary, Dev, or Beta.  If you are not already using one of the Microsoft Edge preview channels, navigate to [Download Microsoft Edge Insider Channels][MicrosoftedgeinsiderDownload] to download one.  
+In the same dropdown, select **Web Browser** > the preview channel of Microsoft Edge that you want Visual Studio to launch, such as Microsoft Edge Canary, Dev, or Beta.  If you are not already using one of the Microsoft Edge preview channels, navigate to [Download Microsoft Edge Insider Channels][MicrosoftedgeinsiderDownload] to download one.  
 
-:::image type="complex" source="./media/set-web-browser.png" alt-text="Choose the preview channel of Microsoft Edge that you want Visual Studio to launch" lightbox="./media/set-web-browser.png":::
-   Choose the preview channel of Microsoft Edge that you want Visual Studio to launch  
+:::image type="complex" source="./media/set-web-browser.png" alt-text="Select the preview channel of Microsoft Edge that you want Visual Studio to launch" lightbox="./media/set-web-browser.png":::
+   Select the preview channel of Microsoft Edge that you want Visual Studio to launch  
 :::image-end:::  
 
-> [!NOTE]
-> If you choose Microsoft Edge \(EdgeHTML\), Visual Studio launches it instead of Microsoft Edge \(Chromium\).  [Install the one of the preview channels of Microsoft Edge][MicrosoftedgeinsiderDownload] or ensure that the version of Microsoft Edge installed on your machine is Microsoft Edge \(Chromium\) and not Microsoft Edge \(EdgeHTML\).  
-
-Now that Visual Studio is correctly configured, choose the green **Play** button.  Visual Studio builds your app, start the web server, launch Microsoft Edge, and navigate to `https://localhost:44362/` or whatever port is specified in `launchSettings.json`.  
+Now that Visual Studio is correctly configured, select the green **Play** button.  Visual Studio builds your app, starts the web server, launches Microsoft Edge, and navigates to `https://localhost:44362/` or whatever port is specified in `launchSettings.json`.  
 
 :::image type="complex" source="./media/edge-launch.png" alt-text="Microsoft Edge launches from Visual Studio" lightbox="./media/edge-launch.png":::
    Microsoft Edge launches from Visual Studio  
 :::image-end:::  
 
 ### Debug JavaScript running in Microsoft Edge  
 
-Switch back to Visual Studio.  In `Counter.js`, to set a breakpoint on Line 13, choose the gutter next to the line.  
+Switch back to Visual Studio.  In `Counter.js`, to set a breakpoint on Line 13, select the gutter next to the line.  
```

Comment:
Depending on how thorough of an update this PR aims for, test the steps, make sure it's still Line 13.  Testing takes a long time, but can yield corrections.  But consider/balance/weigh the opportunity cost of such in-depth testing of steps.

Reply:
Yep, I did test locally, which is how I was able to refresh a couple of the screenshots. The line is still 13, so this is correct.